code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -103,6 +103,53 @@
 	poison_per_bite = 5
 	move_to_delay = 5
 
+//vipers are the rare variant of the hunter, no damage but so much poison medical care will be needed fast. Low hp, high speed and no damage so hit and run tactics.
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter/viper
+	desc = "Furry and black, it makes you shudder to look at it. This one has effervescent purple eyes."
+	icon_state = "hunter"
+	icon_living = "hunter"
+	icon_dead = "hunter_dead"
+	maxHealth = 40
+	health = 40
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	poison_per_bite = 30
+	move_to_delay = 4
+	poison_type = "polonium" //slow acting but this at 30u per bite will seriously mess you up, perfect viper poison
+	speed = 4
+
+//tarantulas are really tanky, regenerating (maybe), hulky monster but are also extremely slow, so.
+/mob/living/simple_animal/hostile/poison/giant_spider/tarantula
+	desc = "Furry and black, it makes you shudder to look at it. This one has abyssal red eyes."
+	icon_state = "guard"
+	icon_living = "guard"
+	icon_dead = "guard_dead"
+	maxHealth = 400 // woah nelly
+	health = 400
+	melee_damage_lower = 35
+	melee_damage_upper = 40
+	poison_per_bite = 0
+	move_to_delay = 8
+	speed = 3

```

Comment:
This is not going to be fun to play as.

Reply:
i could make them fast on webs. oh dear me!